src/main/java/seedu/address/model/contact/NameComparator.java

```diff
@@ -9,15 +9,14 @@
      *
      * @param s1
      * @param s2
-     * @return
+     * @returns a negative, zero or positive integer based on whether the first argument is
+     * less than, equal to or greater than the second.
      */
 
     @Override
     public int compare(Contact s1, Contact s2) {
 
-        Long big = Long.parseLong(s1.getName().fullName, 35);
-        Long next = Long.parseLong(s2.getName().fullName, 35);
-        return big.compareTo(next);
+        return s1.getName().fullName.compareTo(s2.getName().fullName);
```

Comment:
The current implementation treats all capital letters as larger than small letters, which I think is wrong. eg. 'Hope' is sorted above 'arnold' and 'bob':
<img src="https://user-images.githubusercontent.com/60635464/158989180-c75cb623-a777-47a7-b25b-cb6a36d2d0e6.png" width="330" height="399">

Perhaps it might be better to add a `.toLowerCase()` after both the `fullName` so that the order will be 'Amy', 'arnold', 'bob', 'Hope'.